pymongo/topology_description.py

```diff
@@ -253,14 +265,16 @@ def heartbeat_frequency(self) -> int:
     def srv_max_hosts(self) -> int:
         return self._topology_settings._srv_max_hosts
 
-    def _apply_local_threshold(self, selection):
+    def _apply_local_threshold(self, selection: Optional[Selection]) -> List[ServerDescription]:
         if not selection:
             return []
         # Round trip time in seconds.
-        fastest = min(s.round_trip_time for s in selection.server_descriptions)
+        fastest = min(cast(float, s.round_trip_time) for s in selection.server_descriptions)
         threshold = self._topology_settings.local_threshold_ms / 1000.0
         return [
-            s for s in selection.server_descriptions if (s.round_trip_time - fastest) <= threshold
+            s
+            for s in selection.server_descriptions
+            if (cast(float, s.round_trip_time) - fastest) <= threshold
```

Comment:
Hmm I wonder if we could use `float("inf")` as the round trip time for unknown servers rather than None. Could you open a ticket for that idea?

Reply:
yeah!